qp_shogun/sortmerna/tests/test_sortmerna.py

```diff
@@ -0,0 +1,192 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import main
+from os import close, remove, makedirs
+from os.path import exists, isdir, join
+from shutil import rmtree, copyfile
+from tempfile import mkstemp, mkdtemp
+from json import dumps
+from functools import partial
+import os
+
+from qiita_client.testing import PluginTestCase
+
+from qp_shogun import plugin
+from qp_shogun.sortmerna.sortmerna import (
+    generate_sortmerna_commands, sortmerna)
+from qp_shogun.sortmerna.utils import (
+    _per_sample_ainfo, _format_params)
+
+
+SORTMERNA_PARAMS = {
+    'a': 'Number of threads'}
+
+
+class QC_SortmernaTests(PluginTestCase):
+    maxDiff = None
+
+    def setUp(self):
+        plugin("https://localhost:21174", 'register', 'ignored')
+        # db_path = os.environ["QC_SORTMERNA_DB_DP"]
+        self.params = {
```

Comment:
also for completeness, adding here, can you add all parameters, thank you!

Reply:
Added.